src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,15 +1,37 @@
 package core.mate.academy.model;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Add some custom fields that could be only in Bulldozer
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+
+    private int volume;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(int volume) {
+        this.volume = volume;
+    }
+
     @Override
     public void doWork() {
         System.out.println("Bulldozer started to work");
     }
+
+    @Override
+    public List<? extends Machine> get() {
```

Comment:
This method should not be here because this class should not implement interface "MachineProducer". Create class BulldozerProducer for it